src/common/Microsoft.DotNet.UpgradeAssistant.Abstractions/IUpgradeAssistantOptions.cs

```diff
@@ -20,6 +20,6 @@ public class BaseUpgradeAssistantOptions : IUpgradeAssistantOptions
 
         public string? Format { get; set; }
 
-        public UpgradeTarget TargetTfmSupport { get; set; } = UpgradeTarget.LTS;
+        public UpgradeTarget TargetTfmSupport { get; set; } = UpgradeTarget.STS;
```

Comment:
Not sure we want this to be the default. Particularly for upgrade scenarios, customers are more likely to need longer support than new net7 features.

Reply:
@OliaG has specifically asked to change the default to the latest instead of LTS. That is the reason I updated all the defaults. 

@OliaG - Is this decision finalized?

Reply:
As the whole modernization initiative, we want users on the latest released version and encourage them to "Stay current" meaning updating every November. For some workloads like MAUI the difference is very significant.

Plus, those users who want LTS are usually aware of our support policy, terminology etc.  and can specify this parameter explicitly. But by default, users would assume they are getting the latest .NET version.

Happy to chat about it more and loop in more people if you think it's needed. For now, I would keep it STS, the .NETConf is all about porting to .NET 7 :)

Reply:
Yea, I think it makes sense for new project templates to target the latest .NET but my experience with customers *upgrading* is that they're more likely to want LTS (after we've explained the difference - as @twsouthwick points out, they usually don't know how the support policy works). We can chat more offline.

Reply:
There will be additional discussions on this to finalize the long-term plan. For now, will be going ahead with the current change.